### DISCUSSION TITLE
[2기-A] 탁성원

### DISCUSSION THREAD

**Discussion by SeongwonTak** [super-corini/programmers_kdt_II#363] [2기 Monthly Project I]
# Monthly Project 1 : 데이터 시각화 웹 페이지 만들기

## Links
- 웹페이지 : http://taksw222.pythonanywhere.com/
- 블로그기록 (EDA) : https://taksw222.tistory.com/59
- 블로그기록 (배포) : https://taksw222.tistory.com/64
- 블로그기록 (결론) : https://taksw222.tistory.com/65

## Additional Comments
실제로 EDA를 진행하면서 아직은 **데이터를 바라보는 안목이 조금 부족**하다는 느낌이 들어, 앞으로도 EDA나 분석 관련 프로젝트를 Toy Project로 진행하면서, 데이터를 보는 안목을 키우는 것이 가장 중요하다는 생각이 들었습니다.  

HTML, CSS, 배포 등 과정이 쉽지는 않았으나 하나의 사이트를 직접 만들어 보는 과정은 처음이었습니다. 이 쪽에 대해 조금 더 기본적인 지식, 상식을 쌓는 것이 필요하겠다는 생각 또한 들었습니다.

배포나 몇몇 어려운 시각화에서는 구글링을 적극적으로 활용하던가 이도 어려울 경우는 질문하고 부딪혀보면서 해결하였습니다. 특히 시각화에서는 원하는 차트에 대한 구글링 이후 직접 수치 등을 바꿔보며 어떤 효과가 생긴지를 체험해보았습니다.

**Comment by super-corini** [comment]
- E2E는 처음이라 이번 경험은 개발 프로세스의 가장 기초적인 뼈대 부분을 빌드업한다고 생각하시면 좋을 것 같아요. 여기에 좀더 부가적인 것들 (디자인 등)을 붙이는 거겠죠. 
- 평생 공부의 마음으로 꾸준히 학습하시면 됩니다 😎 응원합니당!
